### ISSUE TITLE
Keep track of statistics - Per user

### ISSUE THREAD

**Issue by Zemanzo** [Zemanzo/web-marbles#86]
**Description**
Statistics to track and add to the database per user

**Minimum Acceptance Criteria**
Track:
- [x] Total marbles entered
- [x] Total marbles finished
- [x] Total rounds played
- [x] Total points received

These need to be trackable over time, so saving the timestamp is also required.

**Is blocked by the following issues**
#21

**Comment by Zemanzo** [comment]
No timestamp will be added, instead a copy of the relevant data will be made on a regular basis, and put in another table (where it WILL have a timestamp). This will be added in another issue.
